gulpfile.js

```diff
@@ -182,12 +188,23 @@ const setupCordova = gulp.series(cordovaPlatformAdd, cordovaPrepare, xcode);
 //////////////////
 //////////////////
 
+function preFlightValidation(cb) {
+  log.info(os.platform());
+  if (os.platform() !== 'darwin' && platform === 'ios') {
```

Comment:
macos (`platform == 'osx'`) can also only be built in a macos host.

Reply:
Done in 919eff4

Reply:
Done in 919eff4